pkg/registry/consul.go

```diff
@@ -301,3 +301,9 @@ func (c *ConsulClient) DeleteTree(key string) error {
 	}
 	return nil
 }
+
+// IsAlive checks the health of the consul
+func (c *ConsulClient) IsAlive() (bool, error) {
```

Comment:
add ctx here too, will add jaeger instrumentation on this after the tracing pr is merged